File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/MessagesPage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace gov.va.medora.mdo.domain.sm
{
    [Serializable]
    public class MessagesPage : AbstractPage<Addressee>
    {
        const int _defaultPageSize = 15;
        int _pageSize;

        public MessagesPage()
        {
            _pageSize = _defaultPageSize;
        }

        public int PageSize
        {
            get { return _pageSize; }
            set { _pageSize = value; }
        }
    }
}

File: OtherComponents/MDWSvistalayer/MDWS Source/mdo/mdo/src/mdo/domain/sm/trash/MessagesPage.java
package gov.va.med.mhv.sm.model;

import java.io.Serializable;


public class MessagesPage extends AbstractPage<Addressee> implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = -8357165776736228395L;
	public static final int DEFAULT_PAGE_SIZE = 15;
	
	
	public MessagesPage(){
		this.pageSize = DEFAULT_PAGE_SIZE;
	}
	
	public MessagesPage(int pageSize){
		this.pageSize = pageSize;
	}
		
}
